package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "@exasol/extension-manager-parameter-validator",
```

```suggestion
  "name": "@exasol/extension-parameter-validator",
```
The repo is called differently